worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -149,6 +149,38 @@
         BEFORE_CHANGE
     }
 
+    /**
+     * Reorder mode for {@link EditSession#setReorderMode(ReorderMode)}.
+     *
+     * MULTI_STAGE = Multi stage reorder, may not be great with mods.
+     * FAST = Use the fast mode. Good for mods.
+     * NONE = Place blocks without worrying about placement order.
+     */
+    public enum ReorderMode {
+        MULTI_STAGE("multi"),
+        FAST("fast"),
+        NONE("none");
+
+        private String name;
```

Comment:
Perhaps change this to `displayName`, likewise with `getFromName`? Enums already have a `name` and I think it's a little confusing to have a field with the same name.